runtime/moonbase/src/precompiles.rs

```diff
@@ -92,6 +99,19 @@ parameter_types! {
 
 type EthereumPrecompilesChecks = (AcceptDelegateCall, CallableByContract, CallableByPrecompile);
 
+// Pallet-xcm precompile types.
+// Type that converts AssetId into Location
+type AssetIdToLocationManager = AsAssetType<AssetId, AssetType, AssetManager>;
+
+// The pallet-balances address is identified by 2050
+type SingleAddressMatch = SingleAddressMatcher<AccountId, 2050, Balances>;
+
+// Type that matches an AccountId with a foreign asset address (if any)
+type ForeignAssetMatch = ForeignAssetMatcher<AccountId, AssetId, Runtime, AssetIdToLocationManager>;
+
+// Erc20XcmBridge pallet index = 48
+type Erc20Match = Erc20PalletMatcher<AccountId, 48>;
```

Comment:
Not sure, but we can probably do something like:

```suggestion
type Erc20Match = Erc20PalletMatcher<AccountId, Erc20XcmBridge::index()>;
```

Reply:
We can't do this with the current config of this matcher, because it expects a const and not a `PalletInstance`. Although, I think I will open a PR on Moonkit for this matcher to receive a `PalletInstance` (as the `SingleAddressMatcher` does) and make them consistent between each other.

Reply:
Moonkit PR: https://github.com/Moonsong-Labs/moonkit/pull/44